src/pages/instances/forms/ConvertToInstanceForm.tsx

```diff
@@ -0,0 +1,181 @@
+    formik.resetForm();
+    close();
+  }, [formik.resetForm, close]);
+
+  return (
+    <Modal
+      close={handleCloseModal}
+      className="convert-to-instance-modal"
+      title={
+        uploadState ? "Uploading image to LXD" : "Upload virtual machine image"
+      }
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={handleCloseModal}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={isLoading}
+            disabled={disableSubmit}
+            onClick={() => void formik.submitForm()}
+          >
+            Upload and create
+          </ActionButton>
+        </>
+      }
+    >
+      <Form
+        onSubmit={formik.handleSubmit}
+        className={uploadState ? "u-hide" : ""}
+      >
+        <Input
+          id="image-file"
+          name="imageFile"
+          type="file"
+          label="Image file"
+          onChange={(e) => void handleFileChange(e)}
+        />
+        <Input
+          {...formik.getFieldProps("name")}
+          id="name"
+          type="text"
+          label="New instance name"
+          placeholder="Enter name"
+          error={formik.touched.name ? formik.errors.name : null}
+          disabled={!formik.values.imageFile}
+        />
+        <StoragePoolSelector
+          project={project}
+          value={formik.values.pool}
+          setValue={(value) => void formik.setFieldValue("pool", value)}
+          selectProps={{
+            id: "pool",
+            label: "Root storage pool",
+            disabled: !project || !formik.values.imageFile,
+          }}
+        />
+        <Select
+          {...formik.getFieldProps("architecture")}
+          id="architecture"
+          label="Image architecture"
+          options={supportedVMArchOptions}
+          disabled={!formik.values.imageFile}
+        />
+        <label htmlFor="">Conversion options</label>
+        <div className="conversion-options">
+          <Input
+            {...formik.getFieldProps("formatConversion")}
+            type="checkbox"
+            label={
+              <>
+                Format{" "}
+                <Icon
+                  name="information"
+                  title="Check this option to convert the image into raw format"
+                />
+              </>
+            }
+            disabled={!formik.values.imageFile}
+            checked={formik.values.formatConversion}
+          />
+          <Input
+            {...formik.getFieldProps("virtioConversion")}
+            type="checkbox"
+            label={
+              <>
+                Virtio{" "}
+                <Icon
+                  name="information"
+                  title="Check this option if the image requires Virtio drivers for conversion"
```

Comment:
```suggestion
                  title="Add Virtio drivers to the image. This is needed if..."
```
Again, I am not sure if this is actually true. Can we explain why this should be enabled or disabled?

Reply:
Does this 2nd option depend on the first one to be checked? If so, we should disable it with a title tag stating the dependency in case the 1st option is disabled.

Reply:
You can have both `format` and `virtio` conversion options. Agreed on the label :+1: